worldedit-sponge/src/main/java/com/sk89q/worldedit/sponge/CUIChannelHandler.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.sponge;
+
+import com.sk89q.worldedit.LocalSession;
+import org.spongepowered.api.Platform;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.network.*;
+
+import java.nio.charset.StandardCharsets;
+
+public class CUIChannelHandler implements RawDataListener {
+    public static final String CUI_PLUGIN_CHANNEL = "WECUI";
```

Comment:
Actually, this could be moved to the WorldEdit class probably. However, lets move this in multipass / WorldEdit 7, if someone's trying to use one of the other variables it would be a breaking change.